src/StreamFactoryInterface.php

```diff
@@ -9,15 +9,11 @@
     /**
      * Create a new stream from a resource or a string.
      *
-     * If the argument is a resource, it MUST be readable and SHOULD be seekable. It MAY be writable.
-     * If the argument is a string, a temporary resource will be created that is writable and seekable.
-     * 
-     * If the arugment is a string it will be interpided as the content of the stream. File names or 
-     * file paths are not supported. 
+     * String content will be used as the body of the stream. Files paths are not supported.
```

Comment:
Just to clarify, when I say "User side", I do not mean application users. I mean the developer (library authors) that consume the interface. Example a library author for an API client. 

> Where should I write it?

https://github.com/http-interop/http-factory/pull/16#issuecomment-245717836